reliability/single-fit/trace/src/test/kotlin/com/datadog/android/trace/integration/otel/OtelTracerProviderTest.kt

```diff
@@ -769,7 +770,7 @@ internal class OtelTracerProviderTest {
             }
     }
 
-    @Test
+    @RepeatedTest(10)
```

Comment:
do we have any added value from repeating this test, given that we already have quite large number of spans in the test itself?

Reply:
not quite...you are right, going to lower it.